lib/web_ui/test/engine/view_embedder/dom_manager_test.dart

```diff
@@ -78,6 +78,18 @@ void doTests() {
       // For some reason, only Firefox is able to correctly compute styles for
       // the `::placeholder` pseudo-element.
     }, skip: browserEngine != BrowserEngine.firefox);
+
+    test('removes the outline when focused', () {
```

Comment:
Please move this test to https://github.com/flutter/engine/blob/main/lib/web_ui/test/engine/view_embedder/style_manager_test.dart so you test it directly through `applyGlobalCssRulesToSheet` rather than indirectly through the `DomManager`.

Reply:
I'd argue that it is better in the DomManager. StyleManager and so helpers are good to encapsulate behaviors but one should be able to change them without having to migrate tests like this. In other words, if we kill the style manager in favor of another class that achieves the same goal, we'll likely have to migrate all these tests. If we decide to revamp the way in which we apply styles to the dom manager root element this test here will likeyly stay untouched (good for internal migrations). 

I realized this and kill the `view binding` tests and move the checks where they belong, to the platform dispatcher itself.